python/Ganga/Utility/logging/__init__.py

```diff
@@ -80,7 +59,7 @@ def _get_handlers():
 private_logger = None  # private logger of this module
 
 # main logger corresponds to the root of the hierarchy
-main_logger = _get_logging().getLogger()
+main_logger = logging.getLogger()
```

Comment:
Are we certain that the ```main_logger``` is never multiply defined when the class is imported multiple times?

I had a lot of grief getting the logger to shut down correctly as the final object in the Shutdown manager a while ago (hence a lot of the changes for ```_get_logging``` and I'd like to make sure it gracefully shuts down now)